test/test_encryption.py

```diff
@@ -1105,5 +1107,141 @@ def test_05_endpoint_invalid_host(self):
                 'aws', master_key=master_key)
 
 
+class AzureGCPEncryptionTestMixin(object):
+    KMS_PROVIDER_MAP = None
+    KEYVAULT_DB = 'keyvault'
+    KEYVAULT_COLL = 'datakeys'
+
+    def read_extjson(self, filename):
```

Comment:
We should use the existing `json_data` helper function.

Reply:
I was already using this, just forgot to remove this helper. Done now.